src/Preferences/AdvancedPreferencesViewController.m

```diff
@@ -65,7 +65,8 @@ - (NSString *)toolbarItemLabel
  */
 -(IBAction)showAdvancedHelp:(id)sender
 {
-    GotoHelpPage((CFStringRef)@"advanced.html", NULL);
+    NSString *helpBook = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleHelpBookName"];
+    [[NSHelpManager sharedHelpManager] openHelpAnchor:@"AdvancedSettingsSection" inBook:helpBook];
```

Comment:
OK that wasn't the reason... (at least not directly...)
I suspect this is though:
https://github.com/Eitot/vienna-rss/blob/carbon/configs/scripts/Translations.sh